Misc/NEWS

```diff
@@ -256,6 +256,9 @@ Extension Modules
 Library
 -------
 
+- Issue #29271: Fix Task.current_task and Task.all_tasks implemented in C
```

Comment:
Now Misc/NEWS entries should be prefixed by "bpo-XXXXX" rather of "Issue #XXXXX".